### PR TITLE
Add sitename to structured data

### PR DESCRIPTION
**Pull request by gitstart-sourcegraph**
## Problem Description
Site name displays `Source graph` instead of `Sourcegraph`
Screenshot 2023-03-29 at 12 39 01 PM.

## Description

Adds appropriate sitename to structured data

## Ref
[SG Issue](https://github.com/sourcegraph/about/issues/6139)
[Gitstart Ticket](https://clients.gitstart.com/sourcegraph/1/tickets/SGM-6139)

## Test Plan
Test can be carried out on this change, after deployment and google re-index page.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -240,7 +240,7 @@
     "bundlesize2": "^0.0.31",
     "chalk": "^4.1.0",
     "chokidar-cli": "^2.1.0",
-    "chromatic": "^6.17.2",
+    "chromatic": "^6.11.4",
```
Comment (eseliger) [inline package.json:243]: What's the reason for this change?

Comment (gitstart-sourcegraph) [reply package.json]: This is an unintended change, reverted now.
